.github/NEW_COMPONENT_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+<!--
+Thank you for contributing! Please follow the steps below to help us process your PR quickly.
+
+- 📝 Use a meaningful title for the pull request and include the name of the package modified.
+- 📓 Ensure each of your commit messages adhere to the conventional commit specification.
+- ✅ Add or edit tests to reflect the change (run `yarn test`).
+- 🙏 Please review your own PR to check for anything you may have missed.
+-->
+
+[REPLACE_WITH_JIRA_TICKET_NUMBER](https://oktainc.atlassian.net/browse/REPLACE_WITH_JIRA_TICKET_NUMBER)
+
+## Summary
+
+<!--
+  Add a description with these talking points:
+  1. Figma link if applicable.
+  2. A brief description of the work and why it was done in this particular way.
+-->
+
+## Component checklist
+
+<!-- Pease track and mark relevant items as part of the component development process -->
+
+### Code structure
+
+- [ ] Organize code using a modular component architecture.
+- [ ] Use semantic HTML for clear and meaningful structure.
+- [ ] Break down complex components into smaller, reusable parts.
+
+### Accessibility implementation
+
+- [ ] Apply ARIA attributes to indicate roles and relationships.
+- [ ] Test focus management and ensure interactive elements are keyboard accessible.
+- [ ] Include skip-to-content links for screen reader users.
+- [ ] Provide visual alternatives for auditory elements.
+
+### Internationalization (i18n) implementation
+
+- [ ] Separate text content from the component's code for translation.
+- [ ] Ensure the component renders correctly when language changes.
+
+### Responsive implementation
+
+- [ ] Use CSS media queries to define breakpoints and apply styles.
+- [ ] Test fluid layouts and resizing behavior at each breakpoint as needed.
+- [ ] Optimize images for different screen resolutions as needed.
+
+### Cross-browser compatibility
+
+- [ ] Test the component on major browsers (Chrome, Firefox, Safari, Edge, etc.).
+- [ ] Use browser prefixes or CSS libraries to handle compatibility issues.
+- [ ] Ensure consistent behavior and appearance across browsers.
```

Comment:
When you add these checkboxes, no PR can get merged unless they're all checked.

Is that our intention?

Reply:
My thought was that people could remove irrelevant items from the list to avoid that. I had a comment at the top of the template that was probably a too subtle, I just updated it to say:
`<!-- Pease track and mark relevant items as part of the component development process. Remove any non-applicable items. -->`